packages/react-components/src/organisms/__tests__/Form.test.tsx

```diff
@@ -56,6 +62,81 @@ it('prompts when trying to leave after making edits', () => {
   expect(getUserConfirmation).toHaveBeenCalled();
 });
 
+describe('on cancel', () => {
+  it('prompts after making edits', () => {
+    const { getByText } = render(
+      <ToastContext.Provider value={jest.fn()}>
+        <Router history={history}>
+          <Form {...props} dirty>
+            {({ onCancel }) => (
+              <>
+                <input type="text" required />
+                <Button primary onClick={onCancel}>
+                  cancel
+                </Button>
+              </>
+            )}
+          </Form>
+        </Router>
+      </ToastContext.Provider>,
+      { wrapper: MemoryRouter },
+    );
+
+    userEvent.click(getByText(/^cancel/i));
+    expect(getUserConfirmation).toHaveBeenCalled();
+  });
+  it('goes back in browser history if previous route is available', () => {
```

Comment:
Are the tests reverted? This one is without history and the next one has a history, right?

Reply:
Thanks!